darrencocco/moodle-tool_messagebroker
# Add basic settings

# Testing
1. Browse to "Site administration" > "Plugins" > "Admin tools" > "Message Broker"
2. **Verify** settings display correctly 

Fixes #2
